app/src/main/java/com/jerboa/ui/components/home/HomeActivity.kt

```diff
@@ -228,70 +235,72 @@ fun MainPostListingsContent(
 
     var taglines: List<Tagline>? = null
     when (val siteRes = siteViewModel.siteRes) {
-        ApiState.Loading -> LoadingBar(padding)
-        ApiState.Empty -> ApiEmptyText()
-        is ApiState.Failure -> ApiErrorText(siteRes.msg, padding)
+        ApiState.Loading -> LoadingBar()
+        is ApiState.Failure -> ApiErrorText(siteRes.msg)
         is ApiState.Success -> {
             taglines = siteRes.data.taglines
         }
+
         else -> {}
     }
 
     ReportDrawn()
 
-    val pullRefreshState =
-        rememberPullRefreshState(
-            refreshing = homeViewModel.postsRes.isRefreshing(),
-            onRefresh = {
-                homeViewModel.refreshPosts()
-            },
-        )
+    val pullRefreshState = rememberPullRefreshState(
+        refreshing = homeViewModel.postsRes.isRefreshing(),
+        onRefresh = {
+            homeViewModel.refreshPosts()
+        },
+    )
 
     Box(modifier = Modifier.pullRefresh(pullRefreshState)) {
         // zIndex needed bc some elements of a post get drawn above it.
         JerboaPullRefreshIndicator(
             homeViewModel.postsRes.isRefreshing(),
             pullRefreshState,
             Modifier
-                .padding(padding)
                 .align(Alignment.TopCenter)
                 .zIndex(100f),
         )
         // Can't be in ApiState.Loading, because of infinite scrolling
         if (homeViewModel.postsRes.isLoading()) {
-            LoadingBar(padding = padding)
+            LoadingBar()
         }
 
-        val posts =
+        val posts: List<PostView> = remember(homeViewModel.postsRes) {
```

Comment:
bump on this one too.

Reply:
I'll remove the remember here but it will generate the same one anyway

I could do ApiState<Unit> and expose the posts as `List` then I would be able to directly use it.
I am neutral about that, but I didn't think you would prefer it.